### PR TITLE
Add Redirect for Server Controls

### PR DESCRIPTION
**Pull request by zsarnett**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/ha-panel-config.ts [hunk 1]
```diff
@@ -345,6 +345,8 @@ class HaPanelConfig extends HassRouterPage {
 
   protected routerOptions: RouterOptions = {
     defaultPage: "dashboard",
+    beforeRender: (page) =>
+      page === "server_control" ? "../developer-tools/yaml" : undefined,
```
Comment (balloob) [inline src/panels/config/ha-panel-config.ts:349]: Does `..` work ?

Comment (zsarnett) [reply src/panels/config/ha-panel-config.ts:349]: Seems to in my testing

Comment (bramkragten) [reply src/panels/config/ha-panel-config.ts:349]: We used to do this in the backend: https://github.com/home-assistant/core/blob/dev/homeassistant/components/frontend/__init__.py#L370

Comment (zsarnett) [reply src/panels/config/ha-panel-config.ts:349]: Should we do it there?

**Pull request closed without merging**